### DISCUSSION TITLE
Windows XP

### DISCUSSION THREAD

**Discussion by seba2282** [LegacyUpdate/LegacyUpdate#306] [Questions & Help]
Hello, I test windows XP on Virtualbox from yesterday. And what i saw. It made some updates by Legacy Update and after started work slow and Legacy Update in IE is hanging and can't turn on by X button example. Need put forced shutdown by task manager. You haven't problems something with something with IE in windows all editions in legacy update 1.10 ?

**Comment by kirb** [comment]
I doubt this would be Legacy Update. I’d expect this to more likely be .NET Framework, which runs some recompiling tasks in the background on startup. These can eat a lot of CPU for an extended amount of time - it’s a well-known issue with XP. Do you see mscorsvw.exe running in Task Manager? Do any other processes seem to be using a lot of CPU or memory? Do you perhaps have any IE or Explorer plugins/hacks installed that could be playing up?

Have you also configured the VM with multiple cores, and at least 1GB (ideally more like 2-3GB) of RAM? I believe VirtualBox still defaults to a single core.

**Comment by seba2282** [comment]
I checked all options how you gave here and I tested on 7.1.4 VBOX, now will test on 7.0.22. One core or multicore, XP go extremally in slow. As I saw, Legacy Update in v 1.10 slower finding updates and on vista, 7 and some bootloops, I should see next version Legacy Update in acrtion, to tell if be the same. As I know, you working still over that. So short mean Legacy Update 1.10 on Vbox 7.1.4 was let's download all updates to XP , but made rip when come Embedded updates, didn't can finally connect to download them. 
I don't know what you use to virtualize and check if your project work. But maybe we use other softwares to tests?

**Comment by seba2282** [comment]
I turned off all core isolation in my windows 11 and few started work. General if you will repair all Legacy update and it turn windows update, separate program don't need to install. If someone want old style download, can use, but safer is disable all internet explorer and use example supermium or something other else.  XP is interesting type windows on actually time. I don't know if are some fixes still for him if take that under servers updates how made to Vista/7 now :)
